pkg/parser/phc2sys_parser_test.go

```diff
@@ -52,10 +50,10 @@ func TestPhc2SysParser(t *testing.T) {
 			},
 		},
 		{
-			name:       "Valid regular metrics with sys offset",
-			configName: "ptp4l.0.config",
-			logLine:    "phc2sys[10522413.392]: [ptp4l.0.config:6] CLOCK_REALTIME sys offset 8 s2 freq -6990 delay 502",
+			name:    "Valid regular metrics with sys offset",
+			logLine: "phc2sys[10522413.392]: [ptp4l.0.config:6] CLOCK_REALTIME sys offset 8 s2 freq -6990 delay 502",
 			expectedMetric: &parser.Metrics{
+				ConfigName: "ptp4l.0.config",
```

Comment:
are yo adding configName to metrics attributes now ?

Reply:
Removed